kolibri/core/assets/src/components_demo.js

```diff
@@ -109,6 +109,160 @@ var TextInputWithTagDisplay = Mn.LayoutView.extend({
     }
 });
 
+
+// This is a deeply nested view, used below.
+// Marionette requires deeply-nested views to be defined before they're used in their containing views.
+var ClassroomView = Mn.LayoutView.extend({
+    template: function(serialized_model) {
+        var html = _.join(['<span>', serialized_model.name, ':', '</span>'], '');
+        html += '<ul class="ko_list userList">';
+        _.forEach(this.users, function(user){
+            html += '<li>' +
+                    _.join([user.get('firstname'), user.get('lastname')], ' ') +
```

Comment:
I think we could use some of lodash's embedded model layout reading here instead of constandly appending html:
`<li> <%= user.get('firstname') %>, <%= user.get('lastname') %> </li>'